### PR TITLE
Fix and Version Update to DropletUtils

### PR DESCRIPTION
**Pull request by mtekman**
FOR CONTRIBUTOR:
* [X] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [X] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

@nomadscientist

This fixes the emptyDroplets function which fails after the new filtering mode of STARsolo, and also updates the libs.

### CODE REVIEW

File: tools/dropletutils/dropletutils.xml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
-<tool id="dropletutils" name="DropletUtils" version="@PACKAGE_VERSION@+@GALAXY_VERSION@" >
+<tool id="dropletutils" name="DropletUtils" version="@PACKAGE_VERSION@+galaxy@GALAXY_VERSION@" >
```
Comment (bgruening) [inline tools/dropletutils/dropletutils.xml:2]: Mehmet can you please follow: https://galaxy-iuc-standards.readthedocs.io/en/latest/best_practices/tool_xml.html#tool-versions

Comment (bgruening) [reply tools/dropletutils/dropletutils.xml:2]: Can you please add under xref this bio.tools ID https://bio.tools/dropletutils

Comment (bgruening) [reply tools/dropletutils/dropletutils.xml:2]: And please the EDAM ontologies as well :)

Comment (bgruening) [reply tools/dropletutils/dropletutils.xml:2]: https://docs.galaxyproject.org/en/latest/dev/schema.html#tool-xrefs-xref
https://docs.galaxyproject.org/en/latest/dev/schema.html#tool-edam-topics

Comment (mtekman) [reply tools/dropletutils/dropletutils.xml]: I am not too sure about the edam topics --- I grepped a few keywords but could not find anything single-cell specific, so I opted for general RNA sequencing terms

Comment (bgruening) [reply tools/dropletutils/dropletutils.xml]: The 4 topics and 2 operations are listed here: https://bio.tools/dropletutils

grafik

grafik

Comment (bgruening) [reply tools/dropletutils/dropletutils.xml]: You will find the number when you hover over or click on them.

Comment (mtekman) [reply tools/dropletutils/dropletutils.xml]: Ah thanks! Will fix now

Comment (mtekman) [reply tools/dropletutils/dropletutils.xml]: I got 3 out 4, not bad :D

File: tools/dropletutils/dropletutils.xml [hunk 2]
```diff
@@ -1,9 +1,18 @@
 <?xml version="1.0" encoding="utf-8"?>
-<tool id="dropletutils" name="DropletUtils" version="@PACKAGE_VERSION@+@GALAXY_VERSION@" >
+<tool id="dropletutils" name="DropletUtils" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" >
     <description>Utilities for handling droplet-based single-cell RNA-seq data</description>
+    <xrefs>
+        <xref type="bio.tools">dropletutils</xref>
+    </xrefs>
+    <edam_topics>
```
Comment (bgruening) [inline tools/dropletutils/dropletutils.xml:7]: can you please also add the operations?

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
Merci!

**Pull request merged**